bouncer/src/repo/lib/repo_utils.h

```diff
@@ -60,5 +60,25 @@ namespace repo {
 			char* value = getenv(envVarName.c_str());
 			return (value && strlen(value) > 0) ? value : "";
 		}
+
```

Comment:
Is there no way to detect these error reasons from the write attempt instead of preemptively check for these?

The whole point of this is to try and print the error if possible, if it's not possible I'm not sure if we should bother with this issue.

Reply:
The above code is reverted back as per the discussion.